### PR TITLE
Fixing the title text mismatches in the UI and changing the order of buttons in menu

### PR DESCRIPTION
**Pull request by juusohuhtivuo**
The titles were fixed in both the listItems.js and App.js files. In the listItems.js file, the order of buttons in the menu were changed according to the plan.

### CODE REVIEW

**Comment by juusohuhtivuo** [comment]
First, I accidentally tried to merge this branch with master branch, and ended up closing the wrong pull request. Hopefully that didn't mess anything up.

File: app/src/containers/MenuDrawer/listItems.js [hunk 1]
```diff
@@ -61,27 +46,44 @@ export const secondaryListItems = requestAmount => (
       <ListItemText primary="Find a partner" />
     </ListItem>
 
-    <ListItem button component={Link} to={"/match-requests"}>
```
Comment (janakavishwajith) [inline app/src/containers/MenuDrawer/listItems.js:64]: @juusohuhtivuo I would like to see a visual representation of this list view change. 
Can you upload a screenshot if you don't mind.

Comment (juusohuhtivuo) [reply app/src/containers/MenuDrawer/listItems.js:64]: Sure! Here

UI_menu

Comment (juusohuhtivuo) [reply app/src/containers/MenuDrawer/listItems.js:64]: The order of the buttons was changed according to the plan, but I didn't (at least yet) remove the lines separating buttons.

**Review by Riikka-O — APPROVED** [review]

File: app/src/containers/MenuDrawer/listItems.js (hunk 1 above)
Comment (janakavishwajith) [reply app/src/containers/MenuDrawer/listItems.js:64]: line separation is fine I think
It is separating with a context it seems

Personal Stuffs with messaging
------------------------------------------
Partnering options
------------------------------------------
Redirections and logout

**Review by janakavishwajith — APPROVED** [review]

File: app/src/containers/MenuDrawer/listItems.js (hunk 1 above)
Comment (juusohuhtivuo) [reply app/src/containers/MenuDrawer/listItems.js:64]: We also considered removing the first line completely but then decided not to do that without proper discussion, and the separation seemed to work logically that way, just like you mentioned.

**Review by kmtama — APPROVED** [review]

**Review by tamarameghla — APPROVED** [review]
Good job 👍

**Review by ShocKwav3 — APPROVED** [review]
Well done! I trust you have checked it locally if all the related links/redirects are working.

**Pull request merged**